common/types.go

```diff
@@ -0,0 +1,24 @@
+package common
+
+// JobStatus that represents the Status of a Job
+type JobStatus int
+
+// JobStatus implementation as enum
+const (
```

Comment:
@fforootd

Reply:
this is aligned to the yabs/api, looks good